content/docs/user-guide/components/reference/gradient-modifiers/gradient-mixer.md

```diff
@@ -16,37 +16,61 @@ The **Gradient Mixer** component generates a new gradient by blending a number o
-| **Layers - Gradient** | Refer to [Gradient properties](#gradient-properties) below. | | |
+| **Layers - Enabled** | Toggles the influence of this gradient layer. | Boolean | `Enabled` |
+| **Layers - Operation** | Sets the function that is used to mix this gradient layer with the result of all previous layers. Refer to the [gradient mixing operations](#gradient-mixing-operations) section that follows for more information. | `Initialize`, `Multiply`, `Linear Dodge (Add)`, `Subtract`, `Darken (Min)`, `Lighten (Max)`, `Average`, `Normal`, or `Overlay`. | `Initialize` |
+| **Layers - Gradient** | Refer to the [Gradient properties](#gradient-properties) section that follows. | | |
 
 ### Gradient properties
 
 ![Gradient properties](/images/user-guide/components/reference/vegetation-modifiers/gradient-properties.png)
 
 | Property | Description | Values | Default |
 |-|-|-|-|
-| **Gradient Entity Id** | Sets an entity with an active **Gradient** component. | EntityId | None |
+| **Gradient Entity Id** | Sets an entity with an active **Gradient** or **Gradient Modifier** component. | EntityId | None |
 | **Opacity** | Sets the opacity of the input gradient. | Float: 0.0 - 1.0 | `1.0` |
 | **Invert Input** | Inverts the values of the input gradient. | Boolean | `Disabled` |
-| **Preview (Input)** | Displays the gradient provided by the entity set in **Gradient Entity Id**. |  |  |
+| **Preview (Input)** | Displays the gradient provided by the entity set in **Gradient Entity Id** with the following gradient properties applied. |  |  |
 | **Enable Transform** | If `Enabled`, the translation, scale, and rotation of the input gradient may be modified. | Boolean | `Disabled` |
 | **Translate** | Sets the translation of the input gradient. | Vector3: -Infinity to Infinity | X:`0.0`, Y:`0.0`, Z:`0.0` |
 | **Scale** | Sets the scale of the input gradient. | Vector3: 0.0001 to Infinity | X:`1.0`, Y:`1.0`, Z:`1.0` |
 | **Rotate** | Sets the rotation of the input gradient. | Vector3: -Infinity to Infinity | X:`0.0`, Y:`0.0`, Z:`0.0` |
-| **Enable Levels** | If `Enabled`, the input and output values of the input gradient may be modified. | Boolean | `Disabled` |
-| **Input Mid** | Sets the median value of the input gradient. | Float: 0.0 - 1.0 | `1.0` |
-| **Input Min** | Sets a minimum value for the input gradient. | Float: 0.0 - 1.0 | `0.0` |
-| **Input Max** | Sets a maximum value for the input gradient. | Float: 0.0 - 1.0 | `1.0` |
+| **Enable Levels** | If `Enabled`, the input and output values of the input gradient may be modified by the following properties. | Boolean | `Disabled` |
+| **Input Mid** | Remaps the midpoint of values greater than `0` and less than `1` in the input gradient. | Float: 0.0 - 1.0 | `1.0` |
+| **Input Min** | Remaps the input gradient so that it lies between **Input Min** and **Input Max**.  All output gradient values less than or equal to **Input Min** are set to `0`. | Float: 0.0 - 1.0 | `0.0` |
```

Comment:
Are the _output_ gradient values the ones that are clamped to this minimum? Or should this be referring to the _input_ gradient values? It looks to me like the output gradient values are clamped by Output Min and Output Mix.

Reply:
Yes, this is a hard setting to explain.  It adds a region of `0.0`-valued padding to the end of a gradient.  The Output gradient is not clamped, but _compressed_ between the `0.0`- and `1.0`-valued regions created by **Input Min** and **Input Max**.  Any input gradient could still have the full range of values `0.0` - `1.0`.  Changing the **Output** properties from default _do_ clamp a gradient's values so that it doesn't have a full `0.0` - `1.0` range.  

This group of properties might benefit from their own page with visual examples, I'm hesitant to put examples on every component reference page that uses them.

Reply:
I've added a line to the **Output** properties to point out that the gradient is remapped as opposed to clipped-to-fit.